NFIQ2/NFIQ2Algorithm/src/tool/nfiq2_ui_refresh.cpp

```diff
@@ -779,6 +779,61 @@ void NFIQ2UI::printHeader( NFIQ2UI::Arguments arguments,
   }
 }
 
+std::tuple<std::string, std::string>
+NFIQ2UI::parseModel( NFIQ2UI::Arguments arguments )
+{
+
+  std::string propFilePath;
+
+  if( arguments.flags.model == "" )
+  {
+    // Check common places for directory containing model
+    // If in Current Directory
+    if( BE::IO::Utility::fileExists( "nfiq2rf.txt" ) )
+    {
+      propFilePath = ".";
+    }
+    // Unix
+    else if( BE::IO::Utility::fileExists( "/usr/local/share/nfiq2/nfiq2rf.txt" ) )
+    {
+      propFilePath = "/usr/local/share/nfiq2/";
+    }
+    // Windows
+    else if( BE::IO::Utility::fileExists( "C:\\Program Files\\nfiq2\\nfiq2rf.txt" ) )
+    {
+      propFilePath = "C:\\Program Files\\nfiq2\\";
```

Comment:
Since we're embedding a `/` later on and not using `std::filesystem` yet, let's keep the slashes going forwards. It doesn't necessarily matter.